### PR TITLE
Bug 1766918: Exit the verify_certs loop after 60 iterations

### PR DESCRIPTION
**Pull request by retroflexer**
Closes: #1766918
**- What I did**
Added iteration counter to exit after 60 iterations.
**- How to verify it**
Verify that there is timeout for exit to prevent the script going forever in dead loop
**- Description for the changelog**

timeout verify_certs() to prevent script going forever in dead loop.

### CODE REVIEW

File: templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml [hunk 1]
```diff
@@ -273,8 +273,14 @@ contents:
     }
 
     verify_certs() {
+      iterations=0
       while [ "$(ls $ETCD_STATIC_RESOURCES | wc -l)" -lt 9  ]; do
-        echo "Waiting for certs to generate.."
+        let iterations=$iterations+1
+        if [ $iterations -gt 60 ]; then
```
Comment (hexfusion) [inline templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml]: question:
- How much time does it take for 60 iterations?

- Should the user know about the timeout so they don't kill the script manually? Maybe something like "Waiting N minutes for certs to generate.."

Comment (hexfusion) [reply templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml]: Actually I guess you addressed this

> Waiting for certs to generate... ($iterations/60)

Comment (retroflexer) [reply templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml]: Yeah. Thanks. If there is a different way you want me to tackle it, I can convert it into seconds and minutes too.

Comment (hexfusion) [reply templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml]: 10 mins which I believe is the timeout for the client

**Pull request merged**